src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/DataController.cs

```diff
@@ -146,6 +146,12 @@ public async Task<ActionResult> Get(int instanceOwnerPartyId, Guid instanceGuid,
                 return dataElementError;
             }
 
+            if (!dataElement.IsRead && User.GetOrg() != instance.Org)
```

Comment:
I am a little unsure on this. What if data is shared amongts several orgs like A -melding